packages/plugin-ext/src/hosted/node/plugin-host-rpc.ts

```diff
@@ -80,16 +80,6 @@ export class PluginHostRPC {
         }
     }
 
-    /*
```

Comment:
Hello, why stopContext has been removed ? it's also not referenced in changelog ?


plugin-host-rpc is quite sensitive as it's used downstream like Eclipse Che.

Also I don't see it related to the changes of this PR

Reply:
There were a clean up of plugin unloading that they can be uninstalled and unused clients of refactored code were removed. Code which should be used in a downstream project should be moved to a corresponding project, not kept in the Theia repo without any internal clients and ability to test by any committer.

There is a record that this PR breaks the plugin system. So far we were not listing individual changes only notified that some part will be broken in next release by some PRs, since we did not really document anything as APIs. Please raise it at the dev meeting if you suggest that each a change to each function should be listed.

Please review a PR while it is not merged, not afterwards. It was 9 days while it was in draft mode for the design review. Committers are not expected to review and test PRs against downstream projects or wait when someone does it.

Please also opened follow-up issues to already merged PRs. Not many committers read comments on them and it can get lost.

Reply:
If you're considering that removing code or changing structural behavior is only requiring one approval from a committer of the same company, I think something is wrong.

Reply:
Please add to discuss to the dev meeting.

Regarding to 
> Code which should be used in a downstream project should be moved to a corresponding project, not kept in the Theia repo without any internal clients and ability to test by any committer.

I remembered that we don't have something like DI in the plugin host process yet, not sure whether you can actually extend it. If not please let me know, the issue will be the best.